style.css

```diff
@@ -0,0 +1,525 @@
+    color: white;
+    text-decoration: none;
+    display: block;
+    border-radius: 2px;
+}
+
+.hero__button--transparent:hover {
+    background: white;
+    color: #272626;
+    border-color: white;
+}
+
+.hero__inner {
+    padding-top: 309px;
+    padding-left: 280px;
+    max-width: 584px;
+    max-height: 474px;
+    white-space: normal;
+}
+
+.aside__contact {
+    transform: rotate(-90deg) translate(-45.9%, -657%);
+    background-color: white;
+    position: absolute;
+    min-width: 1080px;
+    max-height: 90px;
+}
+
+.aside__content {
+    padding: 36px 197px 31px 175px;
+    height: 22px;
+    width: 707px;
+    margin: 0px;
+}
+
+.aside__item {
+    font-family: "SofiaPro-Medium";
+    font-size: 16px;
+    letter-spacing: 2.41778px;
+    line-height: 1.4;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.aside__item::before {
+    content: " ";
+    width: 6px;
+    height: 6px;
+    border-radius: 50%;
+    background-color: #55BBA6;
+    margin: 0px 24px;
+    display: inline-block;
+}
+
+.aside__item:first-child::before {
+    content: " ";
+    background-color: white;
+    margin: 0px;
+    height: 0px;
+    width: 0px;
+}
+
+.container {
+    z-index: -1;
+}
+
+.about__container {
+    display: inline-block;
+    position: relative;
+    left: 360px;
+    margin-bottom: 97px;
+}
+
+.about__mission {
+    margin-bottom: 93px;
+}
+
+.about__title {
+    font-family: 'DMSerifDisplay-Regular';
+    font-style: normal;
+    font-weight: normal;
+    font-size: 36px;
+    line-height: 1.36;
+    letter-spacing: 5.44px;
+    color: #55BBA6;
+}
+
+.about__content {
+    display: inline-flex;
+    font-family: "SofiaPro-Regular";
+    font-size: 18px;
+    color: #484848;
+    max-width: 241px;
+    max-height: 120px;
+    margin-right: 116px;
+}
+
+.about__card {
+    width: 487px;
+    height: 601px;
+    margin: 0px;
+    position: relative;
+    background-image: radial-gradient(64.5% 64.5% at 96.09% -12.84%, rgba(0, 0, 0, 0.0001) 0%, rgba(5, 4, 2, 0.6) 100%), url(about-main.jpg);
+    background-position: center;
+    background-size: cover;
+}
+
+.card__content {
+    position: absolute;
+}
+
+.card__content::before {
+    content: '';
+    display: inline-block;
+    height: 100%;
+    vertical-align: middle;
+}
+
+.card__title {
+
+    font-family: "DMSerifDisplay-Regular";
+    font-size: 28px;
+    letter-spacing: 3.76px;
+    color: white;
+    margin-top: 129px;
+    margin-left: 80px;
+}
+
+.card__title::before {
+    content: " ";
+    width: 27px;
+    height: 4px;
+    background-color: #55BBA6;
+    display: block;
+}
+
+.card__info {
+    font-family: "SofiaPro-Regular";
+    font-size: 18px;
+    letter-spacing: 1.02px;
+    color: white;
+    display: inline-flex;
+    max-width: 275px;
+    max-height: 120px;
+    margin-left: 80px;
+    margin-top: 0px;
+}
+
+.card__info::first-line {
+    font-family: "DMSerifDisplay-Regular";
+    color: black;
+
+}
+
+.hero__button--card {

```

Comment:
This belongs to hero. Try to keep your code organized. Keep all classes related to hero where all the other hero elements are styled.